packages/clerk-js/src/ui/components/OrganizationProfile/MembershipWidget.tsx

```diff
@@ -0,0 +1,63 @@
+import { useRouter } from '../../../ui/router';
+import { useCoreOrganization, useOrganizationProfileContext } from '../../contexts';
+import { Col, Flex, Link, Text } from '../../customizables';
+
+export const MembershipWidget = () => {
+  const { organization } = useCoreOrganization();
+  const { __unstable_manageBillingUrl, __unstable_manageBillingMembersLimit } = useOrganizationProfileContext();
+  const router = useRouter();
+
+  if (!organization) {
```

Comment:
Would it be a nice idea to only show this if `__unstable_manageBillingUrl, __unstable_manageBillingMembersLimit` are defined as an extra safety check?